docs/install/kubernetes.md

```diff
@@ -159,20 +159,23 @@ When the installation is complete, the helm command prints the URLs and ports of
 
 ## Configuring the user Jupyter conda environment
 
-Run this in your Jupyter terminal, where `myenv` is the name of your environment:
+The default Jupyter comes with aß conda env named `mlrun`. This conda is installed on a non-persistent path `/opt/conda/envs/mlrun`.
+If you install Python packages on this conda env, and then the Jupyter pod gets deleted or moved to another node, those Python packages will be deleted.
```

Comment:
```suggestion
If you install Python on this conda env, and then the Jupyter pod gets restarted or deleted, those packages will be deleted.
```
The user may have installed other packages, and not just Python, and I think a restart or deletion is clearer than placing the pod in a diffrent node